src/io/sampling.jl

```diff
@@ -370,52 +370,116 @@ function fix_factor!(d::Domain; factors...)::Nothing
 end
 
 """
-    set_factor_bounds!(d::Domain, factor::Symbol, new_bnds::Tuple)::Nothing
-    set_factor_bounds!(d::Domain; factors...)::Nothing
+    get_bounds(dom::Domain, factor::Symbol)::Tuple
 
-Fix bounds of a parameter for sampling to those provided.
+Get factor bounds. If the factor is continuous, it will have a triangular distribution,
```

Comment:
Note, some of the continuous factors have a uniform distribution (e.g. https://github.com/open-AIMS/ADRIA.jl/blob/c07dc5fd16708dd1e8c59a4e4d6ad56745e51c73/src/decision/CriteriaWeights.jl#L12 )

Reply:
Ok, thanks! Question: what determines the type of distribution to be used with each factor?

Reply:
The only descriptor that really captures this is `dists="unif", dists="triag"` etc. It's mostly the coral parameters (defined in `Corals.jl`) and intervention parameters (defined in `Interventions.jl`), that have triangular distributions, but there are a lot of exceptions in there, like `:guided'.